src/style/components/gauge-canvas.module.scss

```diff
@@ -1,9 +1,13 @@
 .root {
-  position: "relative"
+  position: "relative";
+  width: 100%;
+  height: 100%;
 }
 
 .canvas {
   position: absolute;
```

Comment:
Why we use here `position: absolute`?

Reply:
We have two canvases, one for background, one for actual progress - I use this style class for both of them to make sure that they are stacked together

Reply:
ok, it's fine